server/app/views/applicant/IneligibleBlockView.java

```diff
@@ -0,0 +1,51 @@
+package views.applicant;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static j2html.TagCreator.div;
+import static j2html.TagCreator.h1;
+import static j2html.TagCreator.p;
+
+import j2html.tags.specialized.DivTag;
+import java.util.Optional;
+import javax.inject.Inject;
+import play.i18n.Messages;
+import play.mvc.Http.Request;
+import play.twirl.api.Content;
+import views.ApplicationBaseView;
+import views.HtmlBundle;
+import views.style.ApplicantStyles;
+
+public class IneligibleBlockView extends ApplicationBaseView {
+
+  private final ApplicantLayout layout;
+
+  @Inject
+  IneligibleBlockView(ApplicantLayout layout) {
+    this.layout = checkNotNull(layout);
+  }
+
+  public Content render(
+      Request request, String programTitle, Optional<String> applicantName, Messages messages) {
+    // TODO(#3744): Translate these strings.
```

Comment:
reminder (non-blocking): please extract these to the messages files fill in rows in this sheet for them
https://docs.google.com/spreadsheets/d/1x9xq-X61Gev2c470dPYe69tzI4azLTerZu-jfzFHmTE/edit?resourcekey=0-6T7y1Mv3Rr_u1rkmhGCeuQ#gid=0

Reply:
ack, I'll do this since MIriam didn't have time it seems